contracts/package.json

```diff
@@ -13,16 +13,12 @@
     "Solidity",
     "Hardhat"
   ],
-  "author": "",
   "license": "MIT",
   "devDependencies": {
     "@nomicfoundation/hardhat-toolbox": "^2.0.2",
```

Comment:
Why not upgrade `@nomicfoundation/hardhat-toolbox `to the latest 3.0.0 version?

Reply:
@nomicfoundation/hardhat-toolbox@3 is incompatible with the current test cases.

Could you have a try, and analysis if we can upgrade to hardhat-toolbox@3 in another PR?

Reply:
> Could you have a try, and analysis if we can upgrade to hardhat-toolbox@3 in another PR?

Ok, I will upgrade to hardhat-toolbox@3 when I have time.